docs/src/HowToGuides/Atmos/Model/risingbubble.md

```diff
@@ -297,8 +297,8 @@ package](https://CliMA.github.io/CLIMAParameters.jl/latest/)
 A reference state for the linearisation step is also defined.
 
 ```@example risingbubble
-    ref_state =
-        HydrostaticState(DryAdiabaticProfile(typemin(FT), FT(300)), FT(0))
+    T_profile = DryAdiabaticProfile(FT(290))
```

Comment:
```suggestion
    T_profile = DryAdiabaticProfile(FT(300))
```
Leave this at 300 K for this test case. Not sure if that causes the problems, but it might.

Reply:
This is the docs markdown for RB. I can try this change in the risingbuble.jl, though.